js/views/userPageVw.js

```diff
@@ -255,40 +256,55 @@ module.exports = Backbone.View.extend({
       timeout: 4000,
       success: function(model, response){
         //don't render if view has been closed and the $el has been deleted
-        if(self.$el){
-          if (response.profile){
-            $('.js-loadingModal').addClass('hide');
-            if (self.options.ownPage === true){
-              model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash);
-              model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash);
-            }else{
-              model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash + "&guid=" + self.pageID);
-              model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash + "&guid=" + self.pageID);
-            }
+        if (self.isRemoved()) return;
```

Comment:
This is a good way to handle this. There are several views where a fetch takes a while, and a user can navigate away before it's done. At some point we may want to figure out how to make this part of the base view.